docs/usage/etcd.md

```diff
@@ -0,0 +1,63 @@
+# etcd - Key-Value Store for Kubernetes
+
+[etcd](https://etcd.io/) is a strongly consistent key-value store and the most prevalent choice for the Kubernetes
+persistence layer. All API cluster objects like `Pod`s, `Deployment`s, `Secret`s, etc. are stored in `etcd` which
+makes it an essential part of a [Kubernetes control plane](https://kubernetes.io/docs/concepts/overview/components/#control-plane-components).
+
+## Shoot cluster persistence
+
+Each shoot cluster gets its very own persistence for the control plane. It runs in the shoot namespace on the respective
+seed cluster and is managed by a `StatefulSet`. Concretely, there are two `StatefulSets`s (with `volumeClaimTemplates`)
```

Comment:
Looking into https://github.com/gardener/gardener/blob/master/docs/proposals/06-etcd-druid.md, the statefulsets should be abstracted via the druid and the ETCD resource. So, I think from gardener/gardener point of view this documentation must be limited to the ETCD resources and secrets, but implementation details like StatefulSets, volumeClaimTemplates, etc. should be documented in https://github.com/gardener/etcd-druid/.

Reply:
Would it be okay for you if I write this sentence as an implementation **example**? You are right about the abstraction. My goal was also to give the reader an idea about how this is concretely implemented w/o pointing to another documentation. The Druid dependency is hard-coded and well known, so I would say it doesn't hurt.

Reply:
As far as there is no plans to change the implementation behind the ETCD (for example, change the statefulsets to  deployments), I think it is fine.